PostfixCalculator1/PostfixCalculator1.c

```diff
@@ -0,0 +1,84 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include "stack.h"
+#include "postfixCalculator.h"
+
+bool isOperation(char element) {
+    if (element == '+' || element == '-' || element == '*' || element == '/') {
+        return true;
+    }
+    return false;
+}
+
+bool isDigit(char element) {
+    if (element == '0' || element == '1' || element == '2' || element == '3' || element == '4' || element == '5' || element == '6' || element == '7' || element == '8' || element == '9') {
+        return true;
+    }
+    return false;
+}
+
+void takeLastTwoElementsFromTheStack(Stack* stack, int* firstNumber, int* secondNumber) {
+    *firstNumber = top(stack);
+    if (!firstNumber) {
+        return;
+    }
+    pop(stack);
+    *secondNumber = top(stack);
+    if (!secondNumber) {
+        return;
+    }
+    pop(stack);
+}
+
+int postfixCalculator(char* string) {
```

Comment:
```suggestion
int postfixCalculator(const char* string) {
```
Это даст возможность вызывать функцию от строкового литерала